list.js

```diff
@@ -119,6 +119,21 @@
     };
 
     jQuery(document).ready(function(){
+        var boilerplate = `
```

Comment:
If there's no templating going on here, just use normal strings with single-quotes, not back-ticks.